spec/services/people/creator_spec.rb

```diff
@@ -93,30 +89,9 @@
       expect(new_person.ethnicity_id).to eql ethnicity.id
     end
 
-    it 'sets the correct Profile ethnicity' do
-      expect(new_profile.ethnicity_id).to eql ethnicity.id
-    end
-
     it 'sets the correct Person gender' do
       expect(new_person.gender_id).to eql gender.id
     end
-
-    it 'sets the correct Profile gender' do
-      expect(new_profile.gender_id).to eql gender.id
-    end
-
-    it 'does not set the identifiers for the Person' do
```

Comment:
as Person is now the only holder for identifiers, should this test be updated rather than removed?

Reply:
Resolved in latest commit